### ISSUE TITLE
Kiowa using feet from MAP, insterting as METERS

### ISSUE THREAD

**Issue by Scotts556** [aronCiucu/DCSTheWay#170]
Kiowa using feet from MAP, insterting as METERS so when you slave the MMS to a WP is is looking at the sky. or am i doing something wrong?

**Comment by blueberrypi-studio** [comment]
You are correct, I made an error when adding the option to customize altitudes, my initial pull request just used the default altitude, and I somewhat hastily added the option to customize, not realizing it used meters

Will get that fixed
